zcash_client_sqlite/src/lib.rs

```diff
@@ -219,6 +238,17 @@ pub struct WalletDb<C, P> {
     params: P,
 }
 
+impl<C: Borrow<rusqlite::Connection>, P: consensus::Parameters> WalletDb<C, P> {
+    /// Returns the account with the given "one-way stable" identifier, if it was created
+    /// by this wallet instance.
+    pub fn get_account_for_uuid(
+        &self,
+        account_uuid: AccountUuid,
+    ) -> Result<Option<wallet::Account>, SqliteClientError> {
```

Comment:
This is returning (a Result of Option of) a [zcash_client_sqlite::wallet::Account](https://github.com/zcash/librustzcash/blob/2df44970b7510aa0b4d3e82578c374077e7f1f32/zcash_client_sqlite/src/wallet.rs#L199-L205). Why is it returning a `zcash_client_sqlite`-specific type rather than an associated type that impls `zcash_client_backend::data_api::Account`?

I think I'm with @nuttycom here: if for the `zcash_client_sqlite` backend `WalletRead::AccountId` is instantiated as `AccountUuid`, and `WalletRead::Account` is instantiated as `wallet::Account` (as it already is), then the existing `WalletRead::get_account` trait method *is* the required uuid → account mapping.